libr/core/cmd.c

```diff
@@ -153,6 +153,22 @@ static const char *help_msg_star[] = {
 	NULL
 };
 
+static const char *cmd_table_help[] = {
+	"Usage:", ",[/] [file]", "# load table data",
+	",", "", "display table",
+	",", "$foo", "aflt > $foo (files starting with '$' are saved in memory)",
+	", ", " [table-query]", "filter and print table. See ,? for more details",
+	",.", " file.csv", "load the csv file into a table",
```

Comment:
it is not clear that `,.` can be used to load regular files as well, not only .csv